script.module.resolveurl/lib/resolveurl/plugins/drkvid.py

```diff
@@ -24,7 +24,7 @@
 class DrkVidResolver(ResolveGeneric):
     name = 'DrkVid'
     domains = ['drkvid.site', 'allviid.online']
-    pattern = r'(?://|\.)(.*?\.(?:drkvid|allviid)\.(?:site|online))/(?:embed-|play/)?([0-9a-zA-Z]+)'
+    pattern = r'(?://|\.)(.*?\.(?:drkvid|allviid)\.(?:site|online))/(?:embed-|)?([0-9a-zA-Z]+)'
```

Comment:
If you are removing `play/` then you should remove `|` after `embed-` as well
so the line should be
```python
    pattern = r'(?://|\.)(.*?\.(?:drkvid|allviid)\.(?:site|online))/(?:embed-)?([0-9a-zA-Z]+)'
```